packages/eslint-config-hmpps/index.cjs

```diff
@@ -198,6 +212,52 @@ function hmppsConfig({ extraIgnorePaths = [], extraPathsAllowingDevDependencies
         ],
       },
     },
+    // cypress integration tests
+    {
+      files: [`integration_tests/**/*.${scriptExtensionsGlob}`],
+      ...cypressPlugin.configs.recommended,
+    },
+    // non-frontend globals
+    {
+      files: [`**/*.${scriptExtensionsGlob}`],
+      ignores: [`assets/**/*.${scriptExtensionsGlob}`],
+      languageOptions: {
+        globals: {
+          ...globals.node,
+          ...extraGlobals,
+        },
+      },
+    },
+    // unit test globals
+    {
+      files: [`**/*.test.${scriptExtensionsGlob}`],
+      languageOptions: {
+        globals: {
+          ...globals.jest,
+          ...extraUnitTestGlobals,
+        },
+      },
+    },
+    // integration test globals
```

Comment:
could we merge the two cypress configs?

Reply:
i had that initially but it was a bit ugly because of how deeply the globals are nested inside the cypress recommended object:
```typescript
{
  files: [`integration_tests/**/*.${scriptExtensionsGlob}`],
  ...cypressPlugin.configs.recommended,
  languageOptions: {
    ...cypressPlugin.configs.recommended.languageOptions,
    globals: {
      ...cypressPlugin.configs.recommended.languageOptions.globals,
      ...extraIntegrationGlobals,
    },
  },
},
```

so i figured that 2 layers that merge was more readable?

Reply:
might help locating rules that target that same set of files in the same place